gcp/table_gcp_project_organization_policy.go

```diff
@@ -0,0 +1,181 @@
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Constraint").Transform(lastPathElement),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getOrganizationPolicyTurbotData,
+			},
+
+			// standard gcp columns
+			{
+				Name:        "location",
+				Description: ColumnDescriptionLocation,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromConstant("global"),
+			},
+			{
+				Name:        "project",
+				Description: ColumnDescriptionProject,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getProject,
+				Transform:   transform.FromValue(),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listGcpProjectOrganizationPolicies(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create Service Connection
+	service, err := CloudResourceManagerService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+	plugin.Logger(ctx).Trace("listGcpProjectOrganizationPolicies", "GCP_PROJECT: ", project)
+
+	rb := &cloudresourcemanager.ListOrgPoliciesRequest{}
+
+	resp := service.Projects.ListOrgPolicies("projects/"+project, rb)
+	if err := resp.Pages(ctx, func(page *cloudresourcemanager.ListOrgPoliciesResponse) error {
+		for _, orgPolicy := range page.Policies {
+			d.StreamListItem(ctx, orgPolicy)
+		}
+		return nil
+	}); err != nil {
+		return nil, err
+	}
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getGcpProjectOrganizationPolicy(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getGcpProjectOrganizationPolicy")
+
+	// Create Service Connection
+	service, err := CloudResourceManagerService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	project := projectData.Project
+	id := d.KeyColumnQuals["id"].GetStringValue()
+	rb := &cloudresourcemanager.GetOrgPolicyRequest{
+		Constraint: "constraints/" + id,
+	}
+
+	req, err := service.Projects.GetOrgPolicy("projects/" + project, rb).Do()
+	if err != nil {
+		plugin.Logger(ctx).Debug("getGcpProjectOrganizationPolicy", "ERROR", err)
+		return nil, err
+	}
+	return req, nil
+}
+
+func getOrganizationPolicyTurbotData(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+
+	// Get the resource title
+	title := strings.ToUpper(project) + " Org Policy"
```

Comment:
@Priyanka585464 Is this title `required`?
You are not using this `title` value anywhere.
Currently the `title` is using `constraint` value
https://github.com/turbot/steampipe-plugin-gcp/pull/204/files#diff-90f64bc506bbef72770fec8f8ca390942b03e818c0b89357a21e3517f02b28b8R70